Tribler/Test/Core/Modules/RestApi/test_search_endpoint.py

```diff
@@ -0,0 +1,71 @@
+import json
+
+from Tribler.Core.Utilities.twisted_thread import deferred
+from Tribler.Core.simpledefs import NTFY_CHANNELCAST, NTFY_TORRENTS
+from Tribler.Test.Core.Modules.RestApi.base_api_test import AbstractApiTest
+
+
+class TestSearchEndpoint(AbstractApiTest):
+
+    def __init__(self, *args, **kwargs):
+        super(TestSearchEndpoint, self).__init__(*args, **kwargs)
+        self.expected_num_channels = 0
+        self.expected_num_torrents = 0
+
+    def setUp(self, autoload_discovery=True):
+        super(TestSearchEndpoint, self).setUp(autoload_discovery)
+        self.channel_db_handler = self.session.open_dbhandler(NTFY_CHANNELCAST)
+        self.channel_db_handler._get_my_dispersy_cid = lambda: "myfakedispersyid"
+        self.torrent_db_handler = self.session.open_dbhandler(NTFY_TORRENTS)
+
+    def insert_channels_in_db(self, num):
+        for i in range(0, num):
```

Comment:
In this case it probably does not matter qua speed but yeah, it's better practice to use `xrange` for loop ranges.